src/box/xlog.c

```diff
@@ -1357,6 +1360,13 @@ xlog_write_row(struct xlog *log, const struct xrow_header *packet)
 	    xlog_tx_write(log) < 0)
 		return -1;
 
+	struct errinj *inj = errinj(ERRINJ_WAL_WRITE_TIMEOUT, ERRINJ_DOUBLE);
+	if (inj != NULL && inj->dparam > 0) {
```

Comment:
I think you could use `ERROR_INJECT_DOUBLE` here.

Reply:
Done. Change is part of the diff in the first comment of this review iteration.